Libraries/libcar/Headers/car/Rendition.h

```diff
@@ -72,20 +72,49 @@ class Rendition {
         { return _format; }
     };
 
+public:
+    enum resize_mode {
```

Comment:
Yeah, that reasonable. I've already switched to enum class and I'm going through Linux build issues right now.